subsys/bluetooth/Kconfig

```diff
@@ -16,6 +16,12 @@ config BT_MAX_CONN
 	range 1 20 if BT_LL_SOFTDEVICE
 	default 2 if BT_LL_SOFTDEVICE && BT_CENTRAL && BT_PERIPHERAL
 
+config BT_LL_SOFTDEVICE_HEADERS_INCLUDE
```

Comment:
I have two questions here.
What is a use case for having this option?
Why do we want to use headers by default when SD is not used?

Reply:
The headers are used to construct vendor specific commands or parse vendor specific events when the host is not located on the same device as the controller. Example: An application runs on the 53 app core and wants to enable QoS reporting. Then it needs the HCI definitions defined by the SDC

Reply:
Is it ok to have this option enabled when different controller is used? For example BT_LL_SW_SPLIT?

Reply:
That probably does not make sense. We could probably change this to !BT_CTLR